### PR TITLE
Circle graph tooltip content permutations (MNTOR-2693)

### PR DESCRIPTION
**Pull request by flozia**
# References: 
Jira: [MNTOR-2693](https://mozilla-hub.atlassian.net/browse/MNTOR-2693)
Figma: https://www.figma.com/file/CaEKIhvSJqf6KNIMzSkt40/Concepts-for-Monitor-MVP-Redesign?type=design&node-id=2350-93476&mode=design&t=8Uc5sJnqmMW42gbi-4

# Description

Adds the chart tooltip permutations for the “fixed tab” and non-US users.

# How to test

Can be tested in Storybook on the Dashboard pages: `/?path=/story/pages-dashboard`.

# Checklist (Definition of Done)
- [x] Localization strings (if needed) have been added.
- [x] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).
- [x] I've added a unit test to test for potential regressions of this bug.
- [x] Product Owner accepted the User Story (demo of functionality completed) or waived the privilege.
- [x] All acceptance criteria are met.

[MNTOR-2693]: https://mozilla-hub.atlassian.net/browse/MNTOR-2693?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/(proper_react)/redesign/(authenticated)/user/(dashboard)/dashboard/Dashboard.test.tsx [hunk 1]
```diff
@@ -570,6 +571,126 @@ it("toggles between the product offerings in the premium upsell dialog", async (
...
+it("shows chart tooltip on the action needed tab, non-US user", async () => {
+  const user = userEvent.setup();
+  const ComposedDashboard = composeStory(DashboardNonUsNoBreaches, Meta);
+  render(<ComposedDashboard />);
+
+  const chartCaption = screen.getByText(
+    "This chart shows how many times your info is actively exposed.",
+  );
+  expect(chartCaption).toBeInTheDocument();
+  const chartTooltip = within(chartCaption).getByRole("button", {
```
[the remark this answers is not in the document]
Comment (flozia) [reply src/app/(proper_react)/redesign/(authenticated)/user/(dashboard)/dashboard/Dashboard.test.tsx:583]: I’ve just found out about [within](https://testing-library.com/docs/dom-testing-library/api-within/) today as well. It seems to serve pretty much the same purpose: To scope a query to a certain element with a somewhat more descriptive API.

**Review by flodolo — COMMENTED** [review]
(please don't merge without review, especially when strings are exposed for localization)

**Comment by flozia** [comment]
Sorry @flodolo, this was not intentional. Not sure why you were not marked as a codeowner and I was able to merge in the first place. Do you want me to back the PR out of `main`?

**Comment by pdehaan** [comment]
> Not sure why you were not marked as a codeowner and I was able to merge in the first place.

Doesn't GitHub only require ONE codeowner's approval to sign off on a PR to be able to merge? So if @Vinnl gave an approval I don't think GitHub would force you to wait for additional approvals. Maybe&hellip;
Welp, guess I'll be digging through some GitHub docs since that's now a very interesting question.

**Comment by Vinnl** [comment]
image

**Comment by flodolo** [comment]
I think it needs to be [set up](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/about-protected-branches#require-pull-request-reviews-before-merging), but it might not be worth it (sometimes the code owner is not available and you need to override)

> Optionally, you can choose to require reviews from code owners. If you do, any pull request that affects code with a code owner must be approved by that code owner before the pull request can be merged into the protected branch.

Screenshot 2024-01-10 alle 20 55 21

**Comment by pdehaan** [comment]
I know we're getting a bit off topic from the PR, but I don't think this is possible, per https://stackoverflow.com/questions/66821568/require-github-pr-approvals-from-multiple-groups
> As of time of writing, there’s no way to require a review from all code owners assigned to a pull request.

I also trolled through https://github.com/orgs/community/discussions/22522 and https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-file-location and didn't see anything obvious.
Like, we could create a "Team" and assign it as the default repo code owner (similar to how @mozilla/svcops owns **/db/migrations**, which may or may not even be a valid directory these days since I think all migrations are in **https://github.com/mozilla/blurts-server/tree/main/src/db/migrations**). But from the sounds of it, a later match for l10n ftl files (like in our current CODEOWNERS) will override that and take precedence and possibly make @flodolo a required reviewer. But I still think an approval by any of the other devs would make that PR eligible to merge. I think the other solution in one of the threads was to increase the "Required number of approvals before merging" to >1. Which still wouldn't solve the exact problem, and probably make papercut PRs more difficult to merge if we have to chase down approvers to fix docs or typos or a logo.

**Pull request merged**